notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1285 @@
+{
```

Comment:
<p>Can we try to find an image and predicted caption that is better and/or more sensical than this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>I changed it to a simpler baseball image (though I am not sticking to baseball...)</p><p>Also, added another surfing image for a reference.</p>